CHANGES.md

```diff
@@ -1,5 +1,37 @@
 # Changes
 
+## edge-23.8.2
+
+This edge release adds improvements to Linkerd's multi-cluster features as part
+of the [flat network support] planned for Linkerd stable-2.14.0. In addition, it
+fixes an issue ([#10764]) where warnings about an invalid metric were logged
+frequently by the Destination controller.
+
+* Added a new `remoteDiscoverySelector` field to the multicluster `Link` CRD,
+  which enables a service mirroring mod where the control plane
+  performs discovery for the mirrored service from the remote cluster, rather
+  than creating Endpoints for the mirrored service in the source cluster
+  ([#11190], [#11201], [#11220], and [#11224])
+* Fixed missing "Services" menu item in the Spanish localization for the
+  `linkerd-viz` web dashboard ([#11229]) (thanks @mclavel!)
+* Replaced `server_port_subscribers` Destination controller gauge metric with
+  `server_port_subscribes` and `server_port_unsubscribes` counter metrics
+  ([#11206]; fixes [#10764])
+* Replaced deprecated `failure-domain.beta.kubernetes.io` labels in Helm charts
+  with `topology.kubernetes.io` labels ([#11148]; fixes [#11114]) (thanks @piyushsingariya!)
```

Comment:
I think using the full labels `failure-domain.beta.kubernetes.io/zone` and `topology.kubernetes.io/zone` would help people get more context about what this is about.